.gitignore

```diff
@@ -5,3 +5,4 @@
 /settings.json
 /pre-commit-hook.sh
 .vs
+package-lock.json
```

Comment:
package-lock.json shouldn't be in ignore. But you also shouldn't commit package-lock.json unless you actually added some packages. To be sure you're installing with the same version of npm as others, use `meteor npm install`